### PR TITLE
Chartjs

### PR DESCRIPTION
**Pull request by Shreshth3**
-Installed chartjs and react-chartjs-2
-Setup bar charts to display real-time data on frontend

### CODE REVIEW

File: app/components/SystemData.jsx [hunk 1]
```diff
@@ -16,6 +17,52 @@ class SystemData extends Component {
         ? 'None'
         : (sum / numOfDataPoints).toFixed(2);
 
+    const chartData = {
```
Comment (Jbaren01) [inline app/components/SystemData.jsx:20]: is this whats considered "in-line" styling? If so, can it be all done on  .scss file to keep this cleaner? It seems that its a mix...as if this is how chart.js works, but I cant be certain.

File: kafkaApplicationSimulator/kafkaq-config.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const kafkaq = require('../kafkaq-monitor/index.js');
```
Comment (Jbaren01) [inline kafkaApplicationSimulator/kafkaq-config.js]: what is this doing? did we not have/need this before?

**Review by Jbaren01 — APPROVED** [review]
I approved this because I think its correct and necessary to move forward to finish MVP. We can discuss my questions later once we have time.

File: app/components/SystemData.jsx (hunk 1 above)
Comment (Shreshth3) [reply app/components/SystemData.jsx:20]: Good question! It's not in-line styling, but it's very similar. In-line styling would be something like this: [div style="background-color: blue"] Instead, I am setting properties that will go on the Chart object from Chart.js. So, this work must be done in the React component. I like how you were thinking though!

File: kafkaApplicationSimulator/kafkaq-config.js (hunk 2 above)
Comment (Shreshth3) [reply kafkaApplicationSimulator/kafkaq-config.js]: I have no idea how that file got here... I deleted the branch it was on. I will delete the file now. Good catch!

**Pull request merged**